### DISCUSSION TITLE
Grep Blacklist Task

### DISCUSSION THREAD

**Discussion by janvernieuwe** [phpro/grumphp#895] [Ideas]
| Q               | A
| --------------- | ---
| Version         | `grumphp -V`
| Bug?            | no
| New feature?    | yes
| Question?       | no
| Documentation?  | no
| Related tickets | 

Add a grep blacklist task that can work with a whole paths.
This so we can run blacklist checks outside the GitCommit context (i.e. in travis)
This feature should have a list of whitelisted paths to check with an options to blacklist files or paths

This is almost the same command as the current git_blacklist, only it needs additional logic with find and grep -v for the path matching instead of checking the changed files.

**Comment by veewee** [comment]
Since we are allready importing symfony/finder, it might be an option just to check with that component:
https://symfony.com/doc/current/components/finder.html#file-contents

Not sure this one is as fast as grep though.

**Comment by janvernieuwe** [comment]
Finder is probably the better solution for cross-platform compatibility too

**Comment by janvernieuwe** [comment]
WiP https://github.com/janvernieuwe/grumphp/tree/blacklist

**Comment by veewee** [comment]
@janvernieuwe I know it's been a while, but did you get a chance to finish up that task?

**Comment by janvernieuwe** [comment]
No completely overlooked this for a while, its a nice one for hacktoberfest perhaps?
